src/main/jte/view_event.jte

```diff
@@ -0,0 +1,49 @@
+@import edu.kit.hci.soli.domain.Booking
+@import edu.kit.hci.soli.dto.LoginStateModel
+@import edu.kit.hci.soli.domain.ShareRoomType
+@import edu.kit.hci.soli.template.JteContext
+@import edu.kit.hci.soli.template.PageSpec
+@import org.springframework.security.web.csrf.CsrfToken
+@import java.util.Objects
+
+@param JteContext context
+@param LoginStateModel login
+@param Booking booking
+@param CsrfToken csrf
+
+@template.layout(page = new PageSpec("Terminansicht", "Soli"), context = context, login = login, content = @`
+<div class="card-body w-96 mx-auto">
+    <div class="form-control">
+        <label class="label">
+            <span class="label-text">${context.localize("bookings.priority")}</span>
+        </label>
+        <p class="input input-bordered">${context.localize("bookings.priority." + booking.getPriority())}</p>
+    </div>
+    <div class="form-control">
+        <label class="label">
+            <span class="label-text">${context.localize("bookings.description")}</span>
+        </label>
+        <p class="textarea textarea-bordered">${booking.getDescription()}</p>
+    </div>
+    @if(login.kind().equals(LoginStateModel.Kind.ADMIN) || (login.user() != null
+        && Objects.equals(login.user(), booking.getUser())))
+        <form action="/bookings/delete/${booking.getId()}" method="post">
+            @if(csrf != null)
+                <input type="hidden" name="${csrf.getParameterName()}" value="${csrf.getToken()}">
+            @endif
+            <input type="hidden" name="_method" value="delete" />
+            <button type="submit" class="btn btn-ghost fas fa-trash-alt">${context.localize("bookings.delete")}</button>
+        </form>
+    @endif
+    @if(Objects.equals(ShareRoomType.ON_REQUEST, booking.getShareRoomType()) && (login.user() != null
+        && !Objects.equals(login.user(), booking.getUser())))
+        <form action="/bookings/request/${booking.getId()}" method="post">
+            @if(csrf != null)
+                <input type="hidden" name="${csrf.getParameterName()}" value="${csrf.getToken()}">
+            @endif
+            <input type="hidden" name="_method" value="shareRequest" /> <%--This line may cause errors as i dont know what the value should be--%>
```

Comment:
This hidden input is for specifying an HTTP method when doing so otherwise is not possible.
Here, you should just be able to get rid of it.
Since we probably want a separate page where you can enter a message, you can probably even replace this form with a simple link.

Reply:
(Actually, it might make sense for this to redirect users to the booking creation page. We should discuss that tomorrow)